### ISSUE TITLE
pooya's path included in package

### ISSUE THREAD

**Issue by amotarao** [nuxt-community/tailwindcss-module#423]
### Version
@nuxtjs/tailwindcss: v5.0.0-0
nuxt: 

#### dist/module.mjs
https://unpkg.com/@nuxtjs/tailwindcss@5.0.0-0/dist/module.mjs

```mjs
import jiti from 'jiti';
export default jiti(null, { interopDefault: true })('/home/pooya/Code/tailwindcss-module/src/module');
```

#### dist/module.d.ts
https://unpkg.com/@nuxtjs/tailwindcss@5.0.0-0/dist/module.d.ts

```ts
export * from '/home/pooya/Code/tailwindcss-module/src/module';
export { default } from '/home/pooya/Code/tailwindcss-module/src/module';
```

**Comment by 7ochem** [comment]
I tried installing @nuxtjs/tailwindcss at version ^5.0.0-0 and now I'm getting the error:

> ` FATAL  Cannot find module '/home/pooya/Code/tailwindcss-module/src/module'`

**Comment by Atinux** [comment]
Current reproduction: https://stackblitz.com/edit/nuxt-starter-s4qmse?file=nuxt.config.ts

**Comment by pi0** [comment]
Sorry issue should be fixed in `@nuxtjs/tailwindcss@5.0.0-1` release.
